### PR TITLE
Grooming data model

### PR DESCRIPTION
**Pull request by IanMayo**
Ian has taken some steps to groom the datamodel, ready for @theanuradha to start working with it.

### CODE REVIEW

File: info.limpet.stackedcharts.model.tests/src/info/limpet/stackedcharts/model/tests/ModelTests.java [hunk 1]
```diff
@@ -32,6 +32,7 @@ public void testReadModel()
   {
     URI resourceURI = URI.createFileURI("testRead.stackedcharts");
     Resource resource = new ResourceSetImpl().createResource(resourceURI);
+    Assert.assertNotNull("Created resource", resource);
```
Comment (IanMayo) [inline info.limpet.stackedcharts.model.tests/src/info/limpet/stackedcharts/model/tests/ModelTests.java]: @theanuradha - I know this isn't your code.  But, if you get chance to have a look in the next 12 hours, could you find why this resource creation is failing?  The _testRead.stackedcharts_ data-file is present in the plugin, so I can't see why it fails when I run the class as _Junit Test_

thanks,
Ian

**Comment by theanuradha** [comment]
@IanMayo  it works when run as **Plugin-Test**

image

**Comment by theanuradha** [comment]
@IanMayo  as it seems **ResourceSetImpl().createResource(resourceURI)** only works on Plugin context only not on normal Unit test mode.

**Comment by IanMayo** [comment]
Perfect, thanks @theanuradha . Can you remind me how to run the test in a plugin context?

**Comment by theanuradha** [comment]
when you right click on project 
image

**Comment by IanMayo** [comment]
_Ian has deleted some comments related to @theanuradha teaching him to how to run tests_

**Comment by IanMayo** [comment]
Fixes #268

**Pull request merged**